build.js

```diff
@@ -0,0 +1,26 @@
+const {build} = require('esbuild');
+
+const buildOptions = {
+  bundle: true,
+  logLevel: 'info',
+  minify: true,
+  sourcemap: true,
+  outdir: './js',
+  target: ['chrome58', 'firefox57', 'safari11', 'edge16'],
+};
+
+const entryPoint = './scripts/index.js';
```

Comment:
I'm not sure if `scripts` is the right place for it. We have some internal scripts there to help with building stuff, we should not mix it with exportable JS.

I think it would be best to have separate `/js` folder (like our current `/scss` maybe).

In the long term we have to figure out where to keep the source of JS code itself. The fact that we keep it in examples for now is confusing. 

Looking at global-nav, maybe we should end up having `src` folder with the source files for SCSS and JS?

For now let's keep it simple…

We have the `/build` folder, so not to introduce the new ones, maybe we can keep the generated JS there? I think you did that before, right?
Can we then add contents of `/build/js` or something like that into npm package?

Reply:
Here I'm describing the entry point, the file used to build the exported version which will be under `/js/index.js`

If we export the files to the `/build` folder, my understanding is that we'll have to import it like `import {initNavDropdowns} from "vanilla-framework/build/js"` which is a bit strange I think

Reply:
Ah, I've read this wrong. It is entry, not built file, that's fine.

But I think we want the built file to be in the root of project then, or at least in root of npm package, so we can import it directly `from "vanilla-framework"`.

Maybe we can catch up tomorrow, because I'm a bit confused which files are the source, which are built, and what will end up in npm package.

Reply:
That's what I did in the latest changes I'll demo it tomorrow if you want!